dist/images/ovnkube.sh

```diff
@@ -2534,6 +2542,14 @@ ovn-node() {
 
   fi
 
+  if [[ ${ovnkube_node_mode} != "dpu-host" && ! ${ovn_gateway_opts} =~ "gateway-vlan-id" ]]; then
+      # get the gateway vlanid
+      gw_vlanid=$(ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-gw-vlanid | tr -d \")
+      if [[ ${gw_vlanid} != "" ]]; then
```

Comment:
NIT: use -n: `if [[ -n ${gw_vlanid} ]]; then`

Reply:
Fixed this